packages/garbo/src/resources/mayam.ts

```diff
@@ -0,0 +1,262 @@
+/* eslint-disable libram/verify-constants */
+import { cliExecute, Item, myLevel, useFamiliar } from "kolmafia";
+import { $item, clamp, get, maxBy, MayamCalendar } from "libram";
+import { garboValue } from "../garboValue";
+import { GarboTask } from "../tasks/engine";
+import getExperienceFamiliars from "../familiar/experienceFamiliars";
+import { globalOptions } from "../config";
+
+type MayamRingSummon = {
+  choice: string;
+  value: () => number;
+  ring: number;
+};
+
+type Resonance = {
+  rings: string;
+  item: Item;
+};
+
+const RESONANCE: Resonance[] = [
+  { rings: "eye yam2 eyepatch yam4", item: $item`Mayam spinach` },
+  { rings: "vessel yam2 cheese explosion", item: $item`stuffed yam stinkbomb` },
+  { rings: "yam1 meat cheese yam4", item: $item`yam and swiss` },
+  { rings: "yam1 lightning yam3 clock", item: $item`yam battery` },
+  { rings: "yam1 yam2 yam3 explosion", item: $item`thanksgiving bomb` },
+  // {rings: "yam1 meat eyepatch yam4", item: $item`yamtility belt`}
+];
+
+const yamUses = () => get("_mayamSymbolsUsed");
+
+function findResonance(rings: string): Resonance | undefined {
+  return RESONANCE.find((resonance) => resonance.rings === rings);
+}
+
+// Function to find the resonance value
+export function findResonanceValue(input: string): number {
+  const result = findResonance(input);
+
+  // Check if the result is found
+  if (result) {
+    return garboValue(result.item);
+  }
+
+  return 0;
+}
```

Comment:
```suggestion
export function getResonanceValue(combination: MayamCalendar.CombinationString): number {
  const result = MayamCalendar.getResonanceResult(combination);

 if (!result) return 0;
 if (result instanceof Item) {
   if (result === $item`yamtility belt`) return yamtilityValue();
   return garboValue(result);
 }
 return effectValue(result);
}
```
Are all effects of the same duration? or do we need to access that information somewhere?